docroot/themes/custom/mass_theme/templates/views/views-view--data-listing.html.twig

```diff
@@ -86,7 +86,7 @@
     {% endif %}
   {% endset %}
 
-  {% set collection = collection|merge({"content" : collection_content}) %}
+  {% set collection = (collection ?:[])|merge({"content" : collection_content}) %}
```

Comment:
```suggestion
  {% set collection = (collection ?: [])|merge({"content" : collection_content}) %}
```
This didn't get called out, but it seems like the space was intended for after the colon.